debezium-server/debezium-server-core/src/main/java/io/debezium/server/DebeziumMetrics.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.server;
+
+import java.lang.management.ManagementFactory;
+import java.util.Objects;
+
+import javax.enterprise.context.Dependent;
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.DebeziumException;
+import io.debezium.config.CommonConnectorConfig;
+
+/**
+ * Reads debezium source pipeline metrics.
+ * NOTE: calls for reading metrics should be made after debezium connector initialized,
+ * after connector registers metrics, otherwise it will throw `Debezium Mbean not found` error
+ *
+ * @author Ismail Simsek
+ */
+@Dependent
+public class DebeziumMetrics {
+    protected static final Logger LOGGER = LoggerFactory.getLogger(DebeziumMetrics.class);
+    public static final MBeanServer mbeanServer = ManagementFactory.getPlatformMBeanServer();
+    @ConfigProperty(name = "debezium.source.max.queue.size", defaultValue = CommonConnectorConfig.DEFAULT_MAX_QUEUE_SIZE + "")
```

Comment:
PLease insert a newline above this line.
Alsso why is there the DefaultValue caluclated like plus empty String?

Reply:
defaultValue only accepting string value, used "" to convert int to string